Documentation/spelling_wordlist.txt

```diff
@@ -810,6 +812,9 @@ tracepoint
 tracepoints
 tracestate
 transpiling
+Triager
```

Comment:
Nit: sorting in this file is case-sensitive, this entry should go higher up in the file.